docs/reference/pages/panels.rst

```diff
@@ -91,6 +91,18 @@ InlinePanel
 
     This is a powerful but complex feature which will take some space to cover, so we'll skip over it for now. For a full explanation on the usage of ``InlinePanel``, see :ref:`inline_panels`.
 
+
+.. topic:: Collapsing InlinePanels to save space
+
+    By default, ``InlinePanel`` s are expanded and not collapsible. Adding ``collapsible`` to ``classname`` will enable the collapse control. Adding both ``collapsible`` and ``collapsed`` to the ``classname`` parameter will load the editor page with the ``FieldPanel`` collapsed under its heading. You must define a ``heading`` or ``label`` when using ``collapsible`` with ``InlinePanel``.
```

Comment:
@rjmackay This is much better, thanks. Do you think it's worth clarifying that the collapsing is applied to the entire InlinePanel contents, rather than to each individual item?